benchmark/README.md

```diff
@@ -119,6 +119,9 @@ installed libraries instead, enable
 This will reflect the performance of the Swift standard library
 installed on the device, not the one included in the Swift root.
 
+If build fails due with the error like `cannot find X in scope` it my be because default toolchains don't give access to stdlib SPI.
+Try using dev toolchain built locally or dev snapshot from [swift.org](https://www.swift.org/install/).
+
```

Comment:
```suggestion
If a build fails with an error like `cannot find X in scope`, it may be because default toolchains don't give access to stdlib SPI.
Try using a dev toolchain built locally or a dev snapshot from [Swift.org](https://www.swift.org/install/).
```

Reply:
Would this not also better fit into the troubleshooting section?

Reply:
I don't have a strong preference.
Current troubleshooting section is global for all builds. I had to apply those fixes for benchmarks that are built as part of overall build.
This particular advice about toolchain is only applicable to the standalone built via Cmake. This is why I put it here.
LMK if you want me to move it.